workers/typescript/src/payload-converter.ts

```diff
@@ -0,0 +1,58 @@
+import root, { temporal } from './protos/root';
+import {
+  ProtobufBinaryPayloadConverter,
+  ProtobufJsonPayloadConverter
+} from '@temporalio/common/lib/converter/protobuf-payload-converters';
+import {
+  BinaryPayloadConverter,
+  CompositePayloadConverter,
+  JsonPayloadConverter,
+  PayloadConverterWithEncoding,
+  UndefinedPayloadConverter,
+  ValueError
+} from '@temporalio/common';
+import { decode, encode } from '@temporalio/common/lib/encoding';
+import Payload = temporal.api.common.v1.Payload;
+
+export class PassThroughPayload implements PayloadConverterWithEncoding {
+
+  public toPayload(value: any): Payload | undefined {
+    if (!value || value.metadata === undefined || value.data === undefined) {
+      return undefined;
+    }
+    let asPayload;
+    try {
+      asPayload = Payload.fromObject(value as any);
+    } catch (e) {
+      throw new ValueError('PassThroughPayload can only convert Payloads');
+    }
+    const asBytes = Payload.encode(asPayload).finish();
+    return Payload.create({
+      metadata: {
+        encoding: encode(this.encodingType)
+      }, data: asBytes
```

Comment:
can you add a prettier check to verify that code is properly formatted (same goes for linter).

Reply:
Linter is already linting but I'll add a prettier check